### PR TITLE
Change &Proc.new to &block for ruby 2.7 warning

### PR DESCRIPTION
**Pull request by QWYNG**
Hi! thank you for a great gem! 

ruby trank is merged this warning
https://github.com/ruby/ruby/commit/d3da5fbd30e174c5737ec09c6896db7e81691714

I think that Using `&block` is better than `&Proc.new` for Capturing the given block

Hope it makes sense

### CODE REVIEW

File: lib/awesome_print/formatters/base_formatter.rb [hunk 1]
```diff
@@ -110,7 +110,7 @@ def indentation
       end
 
       def indented
```
Comment (HarlemSquirrel) [inline lib/awesome_print/formatters/base_formatter.rb]:
```suggestion
      def indented(&block)
```

File: lib/awesome_print/inspector.rb [hunk 2]
```diff
@@ -63,7 +63,7 @@ def current_indentation
     end
 
     def increase_indentation
```
Comment (HarlemSquirrel) [inline lib/awesome_print/inspector.rb]:
```suggestion
    def increase_indentation(&block)
```

**Comment by QWYNG** [comment]
@HarlemSquirrel  Thank you!

**Comment by vvo** [comment]
@imajes anything required for this PR to be merged? Let us know!

**Review by BryanH — COMMENTED** [review]
These have already been implemented

File: lib/awesome_print/formatters/base_formatter.rb [hunk 3]
```diff
@@ -110,7 +110,7 @@ def indentation
...
-        inspector.increase_indentation(&Proc.new)
+        inspector.increase_indentation(&block)
```
Comment (BryanH) [inline lib/awesome_print/formatters/base_formatter.rb:113]: This has already been implemented

**Comment by BryanH** [comment]
The changes have been implemented as part of the cleanup - thank you very much!

**Pull request closed without merging**